### DISCUSSION TITLE
pydantic 0.28 instead of 0.29

### DISCUSSION THREAD

**Discussion by plankter** [fastapi/fastapi#8162] [Questions] [answered]
`Requires-Dist: pydantic >=0.28,<=0.28.0` is in METADATA file instead of `Requires-Dist: pydantic >=0.28,<=0.29.0` so the pydantic isn't updated.

Meanwhile, `pyproject.toml' file has a valid version range.

**Comment by euri10** [comment]
on a fresh pipenv install I get pydantic 0.29 installed and not the same Requires-Dist
```
➜  fapitest-1K2gCCrj cat ./lib/python3.6/site-packages/fastapi-0.31.0.dist-info/METADATA
Metadata-Version: 2.1
Name: fastapi
Version: 0.31.0
Summary: FastAPI framework, high performance, easy to learn, fast to code, ready for production
Home-page: https://github.com/tiangolo/fastapi
License: UNKNOWN
Author: Sebastián Ramírez
Author-email: tiangolo@gmail.com
Requires-Python: >=3.6
Description-Content-Type: text/markdown
Classifier: License :: OSI Approved :: MIT License
Classifier: Development Status :: 4 - Beta
Classifier: Framework :: AsyncIO
Classifier: Intended Audience :: Developers
Classifier: Programming Language :: Python :: 3.6
Classifier: Programming Language :: Python :: 3.7
Classifier: Programming Language :: Python :: 3.8
Classifier: Programming Language :: Python :: 3 :: Only
Classifier: Topic :: Internet :: WWW/HTTP :: HTTP Servers
Requires-Dist: starlette >=0.11.1,<=0.12.0
Requires-Dist: pydantic >=0.28,<=0.29.0

```

**Comment by tiangolo** [comment, marked as answer]
I'm not sure I undestand the problem.

FastAPI `0.31.0` is compatible with Pydantic `0.28` and `0.29`.

If you have `0.28` installed, that would be used. You can upgrade Pydantic to `0.29` and it would work (it wouldn't complain). If you install fresh, it would install the latest.

**Comment by plankter** [comment]
I think you are right, looks like this is a poetry's issue. It installs not the latest version of pydantic, but version 0.28 even without lock file on a fresh environment.

**Comment by tiangolo** [comment]
Thanks @plankter for reporting back and closing the issue.
